app/containers/wallet/WalletSummaryPage.js

```diff
@@ -20,13 +20,6 @@ const { formattedWalletAmount } = resolver('utils/formatters');
 
 type Props = InjectedProps
 
-const messages = defineMessages({
-  noTransactions: {
-    id: 'wallet.summary.no.transactions',
-    defaultMessage: '!!!No recent transactions',
```

Comment:
Are we okay with removing this given we have no alternative for the classic theme?

Reply:
@SebastienGllmt 
I do not understand how it's related with any Theme.
It's kind of dead code, which is never used.
ATM, we only use `globalMessages.noTransactionsFound`

![image](https://user-images.githubusercontent.com/19986226/58393346-4a739e00-8079-11e9-926c-523af6e695a9.png)

Which was used in a dead block.
![image](https://user-images.githubusercontent.com/19986226/58393373-637c4f00-8079-11e9-9d35-e3470dad8bab.png)